wfl/Controllers/RestaurantsController.cs

```diff
@@ -29,7 +29,7 @@ public RestaurantsController(IDBManager dbManager)
 
         [Route("{id}")]
         [HttpPut]
-        public async Task<IHttpActionResult> UpdateAsync([FromUri] string id, [FromBody] RestaurantDto restaurant)
+        public async Task<IHttpActionResult> UpdateAsync([FromBody] RestaurantDto restaurant, [FromUri] string id)
```

Comment:
Just for my curiosity's sake, why did we reverse the order of these arguments?  Is that the pattern we were following everywhere else?  Was there anything that did not work here?